spec/engine_spec.rb

```diff
@@ -357,6 +358,32 @@
           expect(impressions).to eq([])
         end
       end
+
+      context 'traffic allocations' do
+        before do
+          stub_request(:get, 'https://sdk.split.io/api/splitChanges?since=-1')
+            .to_return(status: 200, body: traffic_allocation_json)
+        end
+
+        it 'returns expected treatment' do
+          expect(subject.get_treatment('01', 'Traffic_Allocation_UI')).to eq(SplitIoClient::Treatments::OFF)
```

Comment:
do we have a case where traffic allocation is 0% and check for default treatment? 
Also the opposite. I saw a test right underneath here but I'm not sure what the default treatment is.